### ISSUE TITLE
Add support for custom build pipeline

### ISSUE THREAD

**Issue by yuzs2** [zilliztech/milvus-operator#56]
Hi, I'm working on milvus-operator deployment in a private cloud environment. To fit our environment, I need to re-edit the milvus dependency helm charts (etcd, minio, pulsar, etc.) and re-build the image. 

To re-build the image, I must customize the Dockerfile and Makefile, and then run `make docker-build`. It's not an elegant way, and it would be very hard to track the latest milvus-operator release with the pipeline. Could you please add the support for custom build in your Makefile? It would be easily to accomplish by adding something like `make docker-build-with-helmcharts {Path}`. Thank you.

**Comment by haorenfsa** [comment]
Yes, it’s a reasonable feature. Would you like to launch a PR yourself?

**Comment by haorenfsa** [comment]
@yuzs2 The version of the dependency charts are maintained in another repo, you can see the history there https://github.com/zilliztech/milvus-helm/commits/master/charts/milvus/charts. Usually we only change them when there're some vulnerability issues or other necessary optimizations. Considering about the stability, the changes will be small, and compatible with previous versions.
